Braintree.podspec

```diff
@@ -50,7 +50,10 @@ Pod::Spec.new do |s|
     s.source_files = "Sources/BraintreeDataCollector/*.swift"
     s.dependency "Braintree/Core"
     s.vendored_frameworks = "Frameworks/XCFrameworks/PPRiskMagnes.xcframework"
-    s.resource_bundle = { "BraintreeDataCollector_PrivacyInfo" => "Sources/BraintreeDataCollector/PrivacyInfo.xcprivacy"}
+    s.resource_bundle = {
+       "BraintreeDataCollector_PrivacyInfo" => "Sources/BraintreeDataCollector/PrivacyInfo.xcprivacy",
+       "PPRiskMagnes_PrivacyInfo" => "Frameworks/XCFrameworks/PPRiskMagnes.xcframework/ios-arm64/PPRiskMagnes.framework/PrivacyInfo.xcprivacy"
+    }
```

Comment:
My concern is that I am not ware that this file path is not relative in some situations, not accessible at the time the pod is created. @scannillo What are your thoughts? Thank you for the suggestion on this.